RESEARCH.md

```diff
@@ -0,0 +1,167 @@
+# Research Assignment: FP in the wild
+
+For this research assignment we choose a programming language, and we take a look at how functional this language. We
```

Comment:
unfinished sentence, missing "is" on the end

Reply:
Fixed in [a38f976](https://github.com/jellehuibregtse/n-queens/pull/1/commits/a38f97626392753bb06fab62e0a44fbafaf57550)